packages/commonwealth/client/styles/components/component_kit/cw_select_list.scss

```diff
@@ -2,4 +2,91 @@
 
 .SelectList {
   @include inputStyles;
+
+  &.isMulti {
+    border: 1px solid $neutral-200 !important;
+    border-radius: 6px !important;
+
+    [class$='-control'] {
```

Comment:
take a look here https://react-select.com/styles
We already use it in `SelectList` to style some components, so would be good to follow this pattern probably.

Reply:
Hm, i initially followed that pattern but before committing moved to all CSS, thinking behind that it will be easy to add responsiveness later on.

Reply:
LMK, what should be the correct approach

Reply:
> what kind of responsiveness?

screen size responsive styles

> Shouldnt it be always 100% width of the parent container, no matter what device size it is?

I think that depends on the usage

> I am more worried about polluting tsx component with a lot of inline styles 🤔 so maybe we can leave it like that for now

same here